### DISCUSSION TITLE
第2节课作业(4班)

### DISCUSSION THREAD

**Discussion by shiruo2686** [InternLM/Tutorial#83] [作业]
4班的第2节课作业，请提交到这里

**Comment by shiruo2686** [comment]
释若的第2节课作业：https://blog.csdn.net/weixin_60280805/article/details/135398779

**Comment by zhjunqin** [comment]
zhjunqin 第二节课作业
https://github.com/zhjunqin/InternLM-tutorial/tree/master/Second

**Comment by helonggood** [comment]
第二节课作业
http://t.csdnimg.cn/Hb8Y6

**Comment by onlyzer0** [comment]
onlyzer0的第二节课作业：https://blog.csdn.net/qq_42907812/article/details/135445094?spm=1001.2014.3001.5501

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/91d7d896cee7443db29f9a355f9e108b?pvs=4
